client/src/components/Workflow/Editor/Index.vue

```diff
@@ -154,6 +145,14 @@
                             @click="undoRedoStore.redo()">
                             <FontAwesomeIcon icon="fa-arrow-right" />
                         </b-button>
+                        <b-button
```

Comment:
Can you make it `disabled` depending on whether there are changes ? I think this was useful behavior in the past.